src/compton_tools/python/csk_reader.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# --------------------------------------*-python-*------------------------------------------------ #
+# file   src/compton_tools/python/csk_reader.py
+# author Andrew Till <till@lanl.gov>
+# date   14 May 2020
+# brief  This script has functions that parse a csk files with Compton
+#        data and return a dense matrix and energy/temperature grids;
+#        If run as executable, saves grids and data with same base filename
+# note   Copyright (C) 2020, Triad National Security, LLC. All rights reserved.
+# ------------------------------------------------------------------------------------------------ #
+
+# ------------------------------------------------------------------------------------------------ #
+# STDLIB
+import os
+import sys
+# TPL
+import numpy as np
+# FPL
+import common_compton as cc
+# ------------------------------------------------------------------------------------------------ #
+
+# These are the functions that are used to read data from the
+# ASCII csk Compton files
+
+
+# ------------------------------------------------------------------------------------------------ #
+def read_csk_files(filebase, verbosity=False):
+    '''Read LANL-style csk file and store into fields and a matrix'''
+
+    # Get path and filePath without extension
+    endings = {'_in_lin', '_out_lin', '_in_nonlin', '_out_nonlin'}
+    fileroot = filebase
+    for ending in endings:
+        if filebase.endswith(ending):
+            fileroot = filebase[:filebase.find(ending)]
+
+    if verbosity:
+        print('Root is', fileroot)
+
+    # normalizations
+    mec2 = 510.998  # keV
+    csk_norm = 2.0 * 0.037558  # currently approximate
```

Comment:
@keadyk Mentioned that this is explained in CSK, do you want to include any of those comments here? Not a big deal.

Reply:
Good call. Added comments and more digits